apps/ensapi/src/config/config.schema.ts

```diff
@@ -42,6 +51,26 @@ export const DatabaseUrlSchema = z.string().refine(
   },
 );
 
+const DateStringToUnixTimestampSchema = z
+  .string()
+  .refine(
+    (dateStr) => {
+      try {
+        const date = new Date(dateStr);
+        return !Number.isNaN(date.getTime());
+      } catch {
+        return false;
+      }
+    },
+    {
+      error: "Invalid date string. Expected a valid date format (e.g., 'Dec 1, 2025 00:00:00 UTC')",
+    },
+  )
+  .transform((dateStr) => {
+    const timestamp = getUnixTime(new Date(dateStr));
+    return makeUnixTimestampSchema().parse(timestamp);
+  });
```

Comment:
Feel free to use ISO 8601 format for defining datetime values in configuration. You can use an existing schema to parse such format:

```suggestion
// you can import `makeDatetimeSchema` from `@ensnode/ensnode-sdk/internal`

const DateStringToUnixTimestampSchema =  z.coerce
    .string()
    .pipe(makeDatetimeSchema())
    .transform((date) => getUnixTime(date));
```